src/backend/database_migrations/versions/20221007_170744_nextclade_output_tables.py

```diff
@@ -0,0 +1,112 @@
+        sa.Column("s3_key", sa.String(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["entity_id"],
+            ["aspen.entities.id"],
+            name=op.f("fk_aligned_peptides_entity_id_entities"),
+        ),
+        sa.ForeignKeyConstraint(
+            ["sample_id"],
+            ["aspen.samples.id"],
+            name=op.f("fk_aligned_peptides_sample_id_samples"),
+        ),
+        sa.PrimaryKeyConstraint("entity_id", name=op.f("pk_aligned_peptides")),
+        sa.UniqueConstraint(
+            "s3_bucket", "s3_key", name="uq_aligned_peptides_s3_bucket_key"
+        ),
+        schema="aspen",
+    )
+    op.create_table(
+        "sample_mutations",
+        sa.Column("id", sa.Integer(), autoincrement=True, nullable=False),
+        sa.Column("sample_id", sa.Integer(), nullable=True),
+        sa.Column("substitutions", sa.String(), nullable=True),
+        sa.Column("insertions", sa.String(), nullable=True),
+        sa.Column("deletions", sa.String(), nullable=True),
+        sa.Column("aa_substitutions", sa.String(), nullable=True),
+        sa.Column("aa_insertions", sa.String(), nullable=True),
+        sa.Column("aa_deletions", sa.String(), nullable=True),
+        sa.ForeignKeyConstraint(
+            ["sample_id"],
+            ["aspen.samples.id"],
+            name=op.f("fk_sample_mutations_sample_id_samples"),
+        ),
+        sa.PrimaryKeyConstraint("id", name=op.f("pk_sample_mutations")),
+        schema="aspen",
+    )
+    op.create_table(
+        "aligned_pathogen_genome",
+        sa.Column("pathogen_genome_id", sa.Integer(), nullable=False),
+        sa.Column("sample_id", sa.Integer(), nullable=False),
+        sa.Column(
+            "aligned_date",
+            sa.DateTime(),
+            server_default=sa.text("now()"),
+            nullable=False,
+        ),
+        sa.Column("reference_name", sa.String(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["pathogen_genome_id"],
+            ["aspen.pathogen_genomes.entity_id"],
+            name=op.f("fk_aligned_pathogen_genome_pathogen_genome_id_pathogen_genomes"),
+        ),
+        sa.ForeignKeyConstraint(
+            ["sample_id"],
+            ["aspen.samples.id"],
+            name=op.f("fk_aligned_pathogen_genome_sample_id_samples"),
+        ),
+        sa.PrimaryKeyConstraint(
+            "pathogen_genome_id", name=op.f("pk_aligned_pathogen_genome")
+        ),
+        sa.UniqueConstraint(
+            "sample_id", name=op.f("uq_aligned_pathogen_genome_sample_id")
+        ),
+        schema="aspen",
+    )
+    op.drop_constraint(
+        "uq_sample_qc_metrics_sample_id",
+        "sample_qc_metrics",
+        schema="aspen",
+        type_="unique",
+    )
+    op.drop_constraint(
+        "fk_sample_qc_metrics_qc_type_qc_types",
+        "sample_qc_metrics",
+        schema="aspen",
+        type_="foreignkey",
+    )
+    op.drop_column("sample_qc_metrics", "qc_type", schema="aspen")
```

Comment:
There's a unique index on sample_qc_metrics.qc_score that doesn't make sense I think? Should we also drop that in this migration?

Reply:
so this was picked up from the auto generate from alembic, i decided to leave it in since it aligns with what's in the sqlalchemy model and we're hooking up these tables to use for real.

Reply:
yeah, this index needs to go away because the qc_type field is going to go away, but there's *another* unique index on the table (not mentioned in this PR) that looks weird to me.

Reply:
oh yeah that's what i was talking about, the sample_qc_metrics.qc_score was picked up from the automigration (even though i didn't touch that in this pr) and i decided to leave it in, I'll take it out though since it's not pertinent to this PR. I was just keeping it in since i'm planning on using that table soon and it seemed like a tiny bit of cleanup to sync up the model code and the DB

Reply:
oh nevermind misread your first comment, i can drop that constraint